applications/RomApplication/python_scripts/python_solvers_wrapper_rom.py

```diff
@@ -37,7 +37,8 @@ def CreateSolverByParameters(model, solver_settings, parallelism):
 
         elif (solver_type == "stationary" or solver_type == "Stationary"):
             solver_module_name = "convection_diffusion_stationary_rom_solver"
-            
+        elif solver_type == "modal_derivative" or solver_type == "Modal_Derivative":
```

Comment:
```suggestion
        elif solver_type == "modal_derivative":
```

to follow the style guide